site/content/en/docs/Integration Patterns/canary-testing.md

```diff
@@ -35,12 +35,17 @@ state that in our system, the label `canary: "true"` will be added to any canary
 apiVersion: "allocation.agones.dev/v1"
 kind: GameServerAllocation
 spec:
-  preferred:
-    - matchLabels:
-        canary: "true"
-  required:
-    matchLabels:
+  - matchLabels:
```

Comment:
@markmandel do you have any preference on if this examples uses the labels `canary: "true"` `canary: "false"` or `agones.dev/fleet: green-fleet` `agones.dev/fleet: blue-fleet`?

```
   selectors: 
     - matchLabels:
         canary: "true"
         agones.dev/fleet: green-fleet
```